flymake-cursor.el

```diff
@@ -143,12 +143,11 @@ the mode directly."
   "pyflake is flakey if it has compile problems, this adjusts the
 message to display, so there is one ;)"
   (cond ((not (or (eq major-mode 'Python) (eq major-mode 'python-mode) t)))
-        ((boundp 'flymake-ler-file)
-         (let ((msg (flymake-ler-file error)))
+        ((boundp 'flymake-diagnostic-text)
```

Comment:
This should be `fboundp`?  My bad the previous code was also wrong 😅

Reply:
yes!  fboundp of course.